### PR TITLE
added deploy to test and prod pipelines

### PR DESCRIPTION
**Pull request by wadeking98**
- Added deploy to test pipeline
- Added deploy to prod pipeline
- Increased resources for web pod 
Signed-off-by: wadeking98 <wkingnumber2@gmail.com>

### CODE REVIEW

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
The deployment strategy you are using for the deployments to `test` and `prod` is meant to find the latest image tag and tag it as a `dev` image, doing the same for `test` and `prod` has the undesired effect of deploying the latest image to the given environment, when what you want to do is "promote" the image that is currently running in the lower level environemnt.  For example when deploying to `test` you want to "promote" the images from the `dev` environment, and when deploying to `prod` you want to "promote" the images from the `test` environment.

Please use the [BC Reg Deployment Pipelines](https://github.com/bcgov/von-bc-registries-agent-configurations/tree/main/jenkins/deployment) as a reference.  If you need something lighter weight refer to the [Family Law Act Deployment Pipelines](https://github.com/bcgov/family-law-act-app/tree/master/jenkins/deployment).  Please follow the same directory structure, which is the same, in the provided references; `jenkins/deployment/<environment>/Jenkinsfile`.

File: jenkins/bc-wallet-demo/Jenkinsfile [hunk 1]
```diff
@@ -32,20 +10,21 @@ String getImageTagHash(String imageName, String tag = "") {
...
+  def config = load "../workspace@script/jenkins/config.groovy"
   stage("Building images ...") {
     script {
       openshift.withCluster() {
         openshift.withProject() {
 
           // Find all of the build configurations associated to the application using labels ...
-          def buildconfigs = openshift.selector("bc", [ app : "${APP_LABEL}" ])
-          echo "Found ${buildconfigs.count()} buildconfigs for app label (app=${APP_LABEL}): ${buildconfigs.names()}"
+          def buildconfigs = openshift.selector("bc", [ app : "${config.APP_LABEL}" ])
```
Comment (WadeBarnes) [inline jenkins/bc-wallet-demo/Jenkinsfile:20]: `config.APP_LABEL` is not defined in `config.groovy`.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Please follow the deployment folder convention:
`jenkins/deployment/<environment>/Jenkinsfile`; for example `jenkins/deployment/test/Jenkinsfile` and `jenkins/deployment/prod/Jenkinsfile`

Plus one other issue to fix.

File: jenkins/bc-wallet-demo/Jenkinsfile (hunk 1 above)
Comment (wadeking98) [reply jenkins/bc-wallet-demo/Jenkinsfile:20]: I think I have it defined: https://github.com/wadeking98/BC-Wallet-Demo-Configurations/blob/31ac7235d64664c3f0ba27f3dcab1a246739dd39/jenkins/config.groovy#L13

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**